umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md

```diff
@@ -38,15 +39,25 @@ the size of the database, ssh into the system and follow the steps below:
 
 Connect to the `umh_v2` or `factoryinsight` database:
 
-```bash
-\c <database-name>
-```
+{{< tabs name="connect_db" >}}
+  {{< tab name="factoryinsight" codelang="sql" >}}
+  \c factoryinsight
+  {{< /tab >}}
+  {{< tab name="umh_v2" codelang="sql" >}}
+  \c umh_v2
+  {{< /tab >}}
+  {{< /tabs >}}
 
  Run the following command to get the size of the database:
 
-```sql
-SELECT pg_size_pretty(pg_database_size('<database-name>'));
-```
+{{< tabs name="get_size_db" >}}
+  {{< tab name="factoryinsight" codelang="sql" >}}
+  SELECT pg_size_pretty(pg_database_size('factoryinsight'));
+  {{< /tab >}}
+  {{< tab name="umh_v2" codelang="sql" >}}
+  SELECT pg_size_pretty(pg_database_size('umh_v2'));
+  {{< /tab >}}
+  {{< /tabs >}}
```

Comment:
This section does not need tabs. After executing the `kubectl exec` command to enter the `psql` shell, the user just need to run the `SELECT pg_size_pretty(pg_database_size('<database-name>'));` query for both databases, as it is possible to get the result from the default database. 
So, after the "This command will open a `psql` shell connected to the default postgres database." sentence, I would just say to execute the query to get the size. Also, it's possible to merge the two queries into one, so that the user can see both sizes at once:
```sql
SELECT pg_size_pretty(pg_database_size('umh_v2')) AS "umh_v2", pg_size_pretty(pg_database_size('factoryinsight')) AS "factoryinsight"
```

Reply:
good to know 👍  updated